airbyte-integrations/connectors/source-s3/metadata.yaml

```diff
@@ -23,8 +23,8 @@ data:
   releases:
     breakingChanges:
       4.0.0:
-        message: "New release using file-based CDK"
-        upgradeDeadline: "2023-08-25"
+        message: "UX improvement and multi-stream support"
```

Comment:
Might be nice to describe what is breaking (for the one user who will actually see this message in production)

Reply:
Good idea, thanks!